coil-base/api/coil-base.api

```diff
@@ -426,6 +427,9 @@ public final class coil/memory/MemoryCache$Key : android/os/Parcelable {
 	public fun writeToParcel (Landroid/os/Parcel;I)V
 }
 
+public final class coil/memory/MemoryCache$Key$Companion {
```

Comment:
I'd rather not have to expose the Companion object, even if empty, but it gets generated regardless

Reply:
Ah yep looks like there's no way around this unfortunately (at least until Kotlin adds namespaces).